.github/workflows/regression.yml

```diff
@@ -128,7 +128,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:220614-1237
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t feature_LM2015 -c develop -s ${{matrix.testscript}}
```

Comment:
@rois1995 for future reference this file can be used to target different branches in Testcases and Tutorials.
Then when things are ready to merge, merge Tutorials/Testcases, change this file back to develop, merge the code PR.

Reply:
Understood, thank you!